doc/api/util.md

```diff
@@ -1014,7 +1521,7 @@ added: v0.6.0
 deprecated: v4.0.0
 -->
 
-> Stability: 0 - Deprecated
+> Stability: 0 - Deprecated: Use [`util.types.isDate()`][] instead.
```

Comment:
`util.types.isDate()` seems undocumented.

Reply:
@vsemozhetbyt Oops, thanks for catching this!